variables.tf

```diff
@@ -92,8 +92,19 @@ variable "global_event_routing_settings" {
     permitted_target_regions  = list(string)
     private_api_endpoint_only = optional(bool, false)
   })
-  description = "Global account settings for event routing. [Learn more](https://cloud.ibm.com/docs/atracker?topic=atracker-settings&interface=ui)"
-  default     = null
+  description = <<EOF
```

Comment:
This description is far too long. Such a description should exist in the example as that is where consumers go to see the usage